src/librawspeed/tiff/TiffIFD.cpp

```diff
@@ -159,7 +160,9 @@ TiffRootIFDOwner TiffIFD::parseMakerNote(NORangesSet<Buffer>* ifds,
     bs.skipBytes(newPosition);
   };
 
-  if (bs.hasPrefix("AOC\0", 4)) {
+  if (bs.hasPrefix("LSI1\0", 5)) { // Skip LSI1 makernotes
```

Comment:
Maybe be more verbose in the comments that this indicates "LaserSoft Imaging" for posterity?

Reply:
If the text of the comment is the only thing that prevents Roman from accepting this patch, then I will certainly change it. Let's wait for his opinion.